src/confluent_kafka/src/Consumer.c

```diff
@@ -1113,13 +1200,33 @@ static PyMethodDef Consumer_methods[] = {
           "  :raises: RuntimeError if called on a closed consumer\n"
 	  "\n"
 	},
+	{ "incremental_assign", (PyCFunction)Consumer_incremental_assign, METH_O,
+	  ".. py:function:: incremental_assign(partitions)\n"
+	  "\n"
+	  "  Incrementally add the provided list of :py:class:`TopicPartition` "
+          "to the current partition assignment.\n"
+	  "\n"
+	  "  :param list(TopicPartition) partitions: List of topic+partitions and optionally initial offsets to start consuming.\n"
+          "  :raises: RuntimeError if called on a closed consumer\n"
```

Comment:
or KafkaException, see unassign() below, and update it for assign() too.

Reply:
yep. i just copy/pasted this from assign.